### PR TITLE
[TD-12157]<enhance>: support json type

### PR DESCRIPTION
**Pull request by huskar-t**
@sangshuduo @zitsen

### CODE REVIEW

File: db/async/row.go [hunk 1]
```diff
@@ -78,7 +78,11 @@ func (a *Async) TaosExec(taosConnect unsafe.Pointer, sql string, timeFormat wrap
 					if row == nil {
 						return nil, FetchRowError
 					}
-					values[j] = wrapper.FetchRow(row, j, rowsHeader.ColTypes[j], lengths[j], precision, timeFormat)
+					v := wrapper.FetchRow(row, j, rowsHeader.ColTypes[j], lengths[j], precision, timeFormat)
+					if vv, is := v.([]byte); is {
+						v = string(vv)
```
Comment (zitsen) [inline db/async/row.go]: One question: how the json data value repr in json response? Raw string or JSON object ?

**Review by zitsen — APPROVED** [review]
LGTM

File: db/async/row.go (hunk 1 above)
Comment (huskar-t) [reply db/async/row.go]: I think it's reasonable to return JSON object. Fixed

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**